index.html

```diff
@@ -0,0 +1,76 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Contact us page</title>
+</head>
+<body>
+    <section>
+        
+    <div class="container">
+        <div class="contactinfo">
+          <div>
+           <h2>Contact info</h2> 
+            <ul class="info">
+                <li>
+                    <span>
+                        <img src="img1/kindpng_1084606.png" width="112px" alt="location"></span>
+                        <span>202 Grant raod<br>
```

Comment:
Why are you using break tag? Always use CSS property for styling eg: max-width: 10ch / warp / flex. Because Address might change, the width might change according to the screen size (repsonsiveness)